analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ParametersCorrectOrder.cs

```diff
@@ -2,6 +2,8 @@
 using System.Collections;
 using System.Collections.Generic;
 using System.Linq;
+using System.Runtime.InteropServices;
+using static Tests.Diagnostics.A;
```

Comment:
Are these needed?

Reply:
Since it's out-of-scope for this PR, it will be addressed in [a new, dedicated PR](https://github.com/SonarSource/sonar-dotnet/pull/8118).